MantidQt/CustomInterfaces/src/Reflectometry/ReflTableViewPresenter.cpp

```diff
@@ -1219,7 +1219,7 @@ void ReflTableViewPresenter::importTable() { m_tableView->showImportDialog(); }
 Export a table to TBL file
 */
 void ReflTableViewPresenter::exportTable() {
-  m_tableView->showAlgorithmDialog("SaveReflTBL");
+  m_tableView->showAlgorithmDialog("SaveTBL");
 }
```

Comment:
Don't forget to do the same for `importTable()`. This is currently done by the view, [here](https://github.com/mantidproject/mantid/blob/15864_generalise_loadReflTBL_and_saveReflTBL/MantidQt/CustomInterfaces/src/Reflectometry/QReflTableView.cpp#L385-L403), we should replace `LoadReflTBL` by `LoadTBL` otherwise the functionality is broken and we can not load TBL files from the interface.